docs/index.html

```diff
@@ -7,3470 +7,1955 @@
+              <li>
+                <p>
+                  the commit message must start with a line containing a
+                  short subject (max 100 characters) which starts by the
+                  <strong>Task issue</strong> number and then describes
+                  briefly what the commit does. For example : TASK-40308
+                  Allow to update kudos message in activity stream
+                </p>
+              </li>
+              <li>
+                <p>
+                  the commit subject must not be a copy/paste of the <strong>Task
+                    issue</strong> summary. As said previously, it must describe
+                  what the commit does, as if the sentence would start
+                  with “If applied, this commit will &#8230;&#8203;”.
+                </p>
+              </li>
+              <li>
+                <p>after the subject, one blank line must be added
+                  before starting the body</p>
+              </li>
+              <li>
+                <p>the body describes in details what the commit
+                  does and must answer 2 questions : why is this change
+                  needed and how does this change address the issue. It
+                  should give as much information as possible to make it
+                  easy for another developer who does not know deeply
+                  the context to understand the commit. Therefore the
+                  body is never too long.</p>
+              </li>
+              <li>
+                <p>after the body, a list of links to relevant
+                  resources can be added (optional). It must be
+                  separated from the body by one blank line.</p>
+              </li>
+            </ul>
+          </div>
+          <div class="paragraph">
+            <p>Here is an example of commit message following these
+              rules:</p>
+          </div>
+          <div class="listingblock">
+            <div class="content">
+              <code class="language-shell hljs" data-lang="shell">TASK-37899
+                Enhance extension registry to trigger event when
+                extensions updated</code>
+              <code class="language-shell hljs" data-lang="shell">
+                When updating extensionRegistry asynchronously from
+                different modules, the other modules should be triggered
+                for this update to allow retrieving newly registered
+                extensions. This way, the extension points in all
+                portlets will can be aware, when listening to adequate
+                event, of any change in extensionRegistry. </code>
+            </div>
+          </div>
+          <div class="paragraph">
+            <p>Of course, all commit messages do not need to be that
+              long, but it must contain all relevant information to
+              understand what and why the changes have been done.</p>
+          </div>
```

Comment:
previous version provide command to create a template, which is proposed when you use command git commit
I found the previous template usefull because he remind each time the model. I think we should keep it

Reply:
It's detailed in github help guides, it takes a lot of place for one notion, this site should be a reference to help guides of third party tools and focus on guidelines instead of a lot of text that will not be read anyway.

Reply:
a reference is ok for me

Reply:
yep, we may need to add an example file in `exo-samples`, like done here:
https://gist.github.com/median-man/3a7c4324005e96f02691f3a20aeac26b